src/content/en/android/trusted-web-activity/play-billing/index.md

```diff
@@ -0,0 +1,142 @@
+project_path: /web/android/_project.yaml
+book_path: /web/android/_book.yaml
+description: Learn how to integrate Google Play Billing into your Trusted Web Activity project.
+
+{# wf_published_on: 2020-11-25 #}
+{# wf_updated_on: 2020-11-25 #}
+{# wf_tags: trusted-web-activity #}
+{# wf_featured_image: /web/updates/images/generic/devices.png #}
+{# wf_blink_components: N/A #}
+
+# Use Play Billing in your Trusted Web Activity {: .page-title }
+
+{% include "web/_shared/contributors/andreban.html" %}
+
+Besides allowing your app to sell digital goods and subscriptions on the Play Store,
+[Google Play Billing][1] offers tools for managing your catalog, prices and subscriptions, useful
+reports, and a checkout flow powered by the Play Store that is already familiar to your users. It
+is also a requirement for applications published on the Play Store that sell digital goods.
+
+Chrome 88 is launching with an Origin Trial on Android that enables the integration of
```

Comment:
Could we have a link on Origin Trial, I'm assuming quite a few people won't know what it is.

Reply:
done